include/libopencm3/stm32/l4/syscfg.h

```diff
@@ -6,16 +6,14 @@
  *
  * @version 1.0.0
  *
- * @date 28 December 2017
+ * @date 2 July 2019
```

Comment:
I'm not convinced this is correct either.  l4/syscfg has definitely existed since dec 2017.